Dockerfile

```diff
@@ -1,10 +1,11 @@
-FROM google/dart-runtime-base:1.25.0-dev.9.0
+FROM google/dart-runtime-base:2.0.0-dev.8.0
 
- # After install we remove the apt-index again to keep the docker image diff small.
- RUN apt-get update && \
-     apt-get upgrade -y && \
-     apt-get install -y git unzip && \
-     rm -rf /var/lib/apt/lists/*
+# After install we remove the apt-index again to keep the docker image diff small.
+RUN apt-get update && \
+    apt-mark hold dart && \
+    apt-get upgrade -y && \
```

Comment:
@isoos – was able to keep upgrade by `holding` dart. Nice!

Reply:
That is much better. We'll get the latest security patches on creation while the dart version is under control, nice!